clang/test/Driver/print-internal-defines-for-sycl.c

```diff
@@ -1,3 +1,4 @@
+// UNSUPPORTED:  i686-pc-windows
```

Comment:
```suggestion
// UNSUPPORTED:  windows
```
Looks like this can be simpler.